chxtools/Circular_Average.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+
+def radial_grid(center, shape, pixel_size=None):
+    """Convert a cartesian grid (x,y) to the radius relative to some center
+
+    Parameters
+    ----------
+    center : tuple
+        point in image where r=0; may be a float giving subpixel precision.
+        Order is (rr, cc).
+    shape : tuple
+        Image shape which is used to determine the maximum extent of output
+        pixel coordinates.
+        Order is (rr, cc).
+    pixel_size : sequence, optional
+        The physical size of the pixels.
+        len(pixel_size) should be the same as len(shape)
+        defaults to (1,1)
+
+    Returns
+    -------
+    r : array
+        The distance of each pixel from `center`
+        Shape of the return value is equal to the `shape` input parameter
+    """
+
+    if pixel_size is None:
+        pixel_size = (1, 1)
+
+    X, Y = np.meshgrid(pixel_size[1] * (np.arange(shape[1]) - center[1]),
+                       pixel_size[0] * (np.arange(shape[0]) - center[0]))
+    return np.sqrt(X*X + Y*Y)
+
+
+def bin_1D(x, y, nx=None, min_x=None, max_x=None):
```

Comment:
same function iskbeam/utils bin_!D?

Why to you need the exact same function from skbeam in chtools? can't to get import from there?